File: trunk/Dobson/DobsonLibrary/Domain/Realm.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace DobsonLibrary.Domain
{
    public class Realm
    {

        public String realm { get; set; }
        public int realmid { get; set; }

        public Realm()
        {
            realm = "";
            realmid = -1;
        }

    }
}

File:  ufmorpheus --username gycordero/OWLMorpheus/src/db/Realm.java
package db;

public class Realm {
	private int realmid;
	private String realm;
	
	public Realm(int id, String realm) {
		this.realmid = id;
		this.realm = realm;
	}
	
	public int getRealmid() {
		return realmid;
	}
	
	public String getRealm() {
		return realm.replaceAll(" ", "_");
	}
}
